.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 build/
 dist/
 npm-debug.log.*
+.idea
```

Comment:
You can use `$HOME/.config/git/ignore` or `$GIT_DIR/info/exclude` to include your editor config files (i use eclipse and my .settings and .project files are there) to avoid populate them in every project